ratelimit/backends/slidingredis.py

```diff
@@ -0,0 +1,110 @@
+import datetime
+import json
+import logging
+import time
+
+from aredis import StrictRedis
+
+from ..rule import Rule
+from . import BaseBackend
+
+
+class TimeFilter(logging.Filter):
+    def filter(self, record):
+        try:
+            last = self.last
+        except AttributeError:
+            last = record.relativeCreated
+        delta = datetime.datetime.fromtimestamp(
+            record.relativeCreated / 1000.0
+        ) - datetime.datetime.fromtimestamp(last / 1000.0)
+        record.relative = "{0:.2f}".format(
+            delta.seconds + delta.microseconds / 1000000.0
+        )
+        self.last = record.relativeCreated
+        return True
+
+
+logger = logging.getLogger("ratelimit")
+logger.setLevel(logging.DEBUG)
+ch = logging.StreamHandler()
+ch.setLevel(logging.DEBUG)
+formatter = logging.Formatter(
+    fmt="+%(relative)s - %(name)s - %(levelname)s - %(message)s"
+)
+logger.addHandler(ch)
+[hndl.addFilter(TimeFilter()) for hndl in logger.handlers]
+[hndl.setFormatter(formatter) for hndl in logger.handlers]
```

Comment:
what to do with this, comment it here or paste it in the PR so that it is useful to someone else down the road. It spits out a nice logging output so that you can see time spent between 2 log events, this avoids headaches when trying to figure out if yes/no a request should be blocked:
```
+0.00 - ratelimit - DEBUG - 1616057360.867704 [0, 2] : False
+1.01 - ratelimit - DEBUG - 1616057361.8722386 [1, 1] : True
+0.00 - ratelimit - DEBUG - 1616057361.876414 [0, 0] : False
+2.01 - ratelimit - DEBUG - 1616057363.8830643 [1, 0] : False
```

Reply:
Maybe putting it in tests would be a better choice.

Reply:
definitely yes ! I didnt think about it